src/components/posts/PostsPage.js

```diff
@@ -39,29 +40,36 @@ const PostsPage = ({ role }) => {
 
   return (
     <>
-      <section className='posts-section'>
-        {postResponse.status}
-        <PostForm handleSubmit={createPost} handleChange={handleChange} inputValue={post.content} />
-        <ul className="posts-list">
-          {posts.map((post, index) => (
-            <li key={index} className='post-item'>
-              <Box>
-                <div className="post-content">{post.content}</div>
-                <Stack className="names-date" spacing={2} direction="row">
-                  <Link to={`/user/${post.user.id}`} className='post-author'>
-                    <Box className="fullname" variant='contained'>
-                      <strong>{`${post.user.profile.firstName} ${post.user.profile.lastName}`}</strong>
+      <div className='home-container'>
```

Comment:
I'd reccomend perhaps restructuring this component now to a more general "Home" component - it feels a bit strange for the Cohort list to be inside a "Posts" component - it's more like we have a "Home" component that contains post and a cohort list.